doc/modules/model_evaluation.rst

```diff
@@ -555,7 +555,8 @@ by varying a decision threshold.
 
 The :func:`average_precision_score` function computes the average precision
 (AP) from prediction scores. This score corresponds to the area under the
-precision-recall curve.
+precision-recall curve. The value is between 0 and 1, that higher is better.
```

Comment:
Thanks.  Sorry for my ignorance. Anything else which might to be good to add here ?